tests/test_kubernetes_tools.py

```diff
@@ -1902,6 +1903,16 @@ def test_update_deployment():
     )
 
 
+@mock.patch("paasta_tools.kubernetes_tools.KubernetesDeploymentConfig", autospec=True)
+def test_set_instances_for_kubernetes_service(mock_kube_deploy_config):
+    replicas = 5
+    mock_client = mock.Mock()
+    mock_kube_deploy_config.get_sanitised_deployment_name.return_value = 'fake_deployment'
+    mock_kube_deploy_config.format_kubernetes_app.return_value = mock.Mock()
+    set_instances_for_kubernetes_service(mock_client, mock_kube_deploy_config, replicas)
+    assert mock_client.deployments.patch_namespaced_deployment_scale.call_count == 1
```

Comment:
nit: You should also check the args for the `patch_namespace_deployment_scale` to make sure the proper replica value is passed in.